packages/eslint-plugin-pf-codemods/src/rules/v6/textReplaceWithContent/text-replace-with-content.ts

```diff
@@ -71,11 +71,28 @@ module.exports = {
               const specifierName = specifierToReplace.imported.name;
               const newText = getNewText(specifierName);
 
+              const importPath = node.source.value?.toString() as string;
```

Comment:
Is there a way to do this without the type assertion? If not is this type assertion truly safe?

It seems like `importPath` could potentially resolve as undefined here.

Reply:
I mean, technically it should always be a string starting with `"@patternfly/react-core"`, but I'll update it so there is not the casting